code/_experiments.dm

```diff
@@ -3,32 +3,20 @@
 // Any flag you see here can be flipped with the `-D` CLI argument.
 // For example, if you want to enable EXPERIMENT_MY_COOL_FEATURE, compile with -DEXPERIMENT_MY_COOL_FEATURE
 
-// EXPERIMENT_515_QDEL_HARD_REFERENCE
-// - Hold a hard reference for qdeleted items, and check ref_count, rather than using refs. Requires 515+.
-
-// EXPERIMENT_515_DONT_CACHE_REF
-// - Avoids `text_ref` caching, aided by improvements to ref() speed in 515.
+// EXPERIMENT_MY_COOL_FEATURE
+// - Does something really cool, just so neat, absolutely banging, gaming and chill
 
 #if DM_VERSION < 515
 
-// You can't X-macro custom names :(
-#ifdef EXPERIMENT_515_QDEL_HARD_REFERENCE
-#warn EXPERIMENT_515_QDEL_HARD_REFERENCE is only available on 515+
-#undef EXPERIMENT_515_QDEL_HARD_REFERENCE
-#endif
-
-#ifdef EXPERIMENT_515_DONT_CACHE_REF
-#warn EXPERIMENT_515_DONT_CACHE_REF is only available on 515+
-#undef EXPERIMENT_515_DONT_CACHE_REF
-#endif
-
+	// You can't X-macro custom names :(
+	#ifdef EXPERIMENT_MY_COOL_FEATURE
+		#warn EXPERIMENT_MY_COOL_FEATURE is only available on 515+
+		#undef EXPERIMENT_MY_COOL_FEATURE
+	#endif
```

Comment:
fyi macros being unscoped is common in preprocessed languages in general to not imply they have any actual scoping. doesnt matter so much here since there actually is scope here to worry about but just saying it wasnt a mistake the first time

Reply:
I forgot how I like macros indented, remember now, thanks

Reply:
ah right no I wrote it how I like it. you get it, we good